src/components/Shared/Attachment.tsx

```diff
@@ -78,26 +79,85 @@ const Attachment: FC<Props> = ({ attachments, setAttachments }) => {
 
   return (
     <div>
```

Comment:
**question:** do we need div here? also is this matching with the same design while we select reference module?

Reply:
No, here the Menu.Item wrapper is removed because onClick menu item, the popover closes which stops onChange event of input. (because it got removed from dom)